IPs/RSKIP454.md

```diff
@@ -0,0 +1,59 @@
+---
+rskip: 454
+title: Support bitcoin blocks with chain work up to 32 unsigned bytes
+created: 25-Oct-24
+author: NC
+purpose: Sca
+layer: Core 
+complexity: 1
+status: Draft
+description: 
+---
+
+|RSKIP          | 454                                                                   |
+| :------------ |:----------------------------------------------------------------------|
+|**Title**      | Support bitcoin blocks with chain work up to 32 unsigned bytes |
+|**Created**    | 25-Oct-24                                                             |
+|**Author**     | NC                                                                    |
+|**Purpose**    | Sca                                                                   |
+|**Layer**      | Core                                                                  |
+|**Complexity** | 1                                                                     |
+|**Status**     | Draft                                                                 |
+
+## Abstract
+
+This RSKIP proposes enabling RSK to accept Bitcoin blocks with chain work up to 32 unsigned bytes (256 bits). This change is necessary to ensure the bridge can process Bitcoin blocks with chain work values that exceed the current 12 unsigned bytes limit (96 bits).
+
+Before [RSKIP434](RSKIP434.md) the chain work was even lower (12 signed bytes, 1 bit reserved for the sign and 95 bits used to store the chain work value). This 95 bits limit chain work got surpassed by the block #849,138 causing rsk nodes to be unable to process Bitcoin blocks, resulting in a [Rootstock peg-in / peg-out service outage](https://blog.rootstock.io/noticia/incident-report-rootstock-peg-in-peg-out-service-outage-on-june-24th/).[1]
```

Comment:
Perhaps RSKIP434 should also be put as a reference?

Reply:
Done